layouts/partials/captivate/main_channel.html

```diff
@@ -1,6 +1,21 @@
+{{ $channel := index $.Site.Data.channels (printf "x%s" .Params.channelid ) }}
 <main id="main-content" class="main-content">
-    <div class="container">
-        <div class="row">
+    <section class="channel-page-banner" style="background-image: url('/{{.Type}}/images/channel/{{.Params.channelid}}-banner.jpg');">
```

Comment:
where is this channel banner stored? Is it coming from the database? what happens if the channel is replaced?